en/docs/reference/connectors/file-connector/file-connector-config.md

```diff
@@ -2931,3 +2931,157 @@ The following operations allow you to work with the File Connector version 4. Cl
        <detail>Target file already exists. Path = file:///Users/hasitha/temp/file-connector-test/copy/kandy/hasitha.txt</detail>
     </writeResult>
     ```
+??? note "fetchDirectoryContent"
```

Comment:
Hi @dulanjalidilmi, 
The content under `???note` should be indented by [four spaces](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage). Otherwise, the note block will not render correctly.

Reply:
Fixed. Thank you very much